src/main/js/helpers/legend.js

```diff
@@ -70,15 +70,17 @@ const loadLegendItem = (legendSVG, t, shownTargets, eventHandlers) => {
     validateLegendLabel(t.label);
     const text = getText(t.label.display);
     const index = shownTargets.indexOf(t.key);
+    const disabled =
+        !!t.label.isDisabled || (!!t.values && t.values.length == 0);
```

Comment:
You can use `utils.isEmpty()` here? You already have validated it to be an array during Validation Phase anyway.